vtysh/bgp_vty.c

```diff
@@ -5182,13 +5229,21 @@ DEFUN(no_neighbor_timers,
     get_bgp_neighbor_with_bgp_router_and_ipaddr(bgp_router_context, ip_addr);
 
     if (ovs_bgp_neighbor) {
-        key_timers[0] = "Keepalive";
-        key_timers[1] = "Holdtimer";
-        tim_val.keepalive = 0;
-        tim_val.holdtime = 0;
-        /* To write to ovsdb nbr table. */
-        ovsrec_bgp_neighbor_set_timers(ovs_bgp_neighbor, key_timers,
-                                       (int64_t *)&tim_val,0);
+        if (ovs_bgp_neighbor->bgp_peer_group) {
+	    VLOG_ERR(OVSDB_TXN_COMMIT_ERROR);
+            VLOG_ERR("Error in transaction for setting timers of neighbor."
+                     "Neighbor %s already has been assigned to the peer group.\n",
+                                                                        argv[0]);
```

Comment:
propose exclude macros VLOG_ERR.. Logging with level of DBG is already in ERRONEOUS_DB_TX macro.